### PR TITLE
Feature/add reason react template

### LINKED ISSUE #5: Add hygen templates
- [x] Reason React

### PR DESCRIPTION
**Pull request by jrquadros**
closes #5

### CODE REVIEW

**Comment by fakenickels** [comment]
We'll keep bs-universe only for bindings

**Comment by guilhermedecampo** [comment]
@fakenickels 	yeah but binding also react libs no?

**Comment by fakenickels** [comment]
aaaah indeed lol sorry the confusion @jrquadros

File: newLib.js [hunk 1]
```diff
@@ -5,13 +5,15 @@ const projectName = process.argv.slice(2).toString()
...
+
 async function execute(command) {
   const logs = await execAsync(command)
   process.stdout.write(logs.stdout)
 }
 
 const main = async () => {
-  await execute(`npx hygen lib new ${projectName}`)
+  await execute(`npx hygen ${isReact} new ${projectName}`)
   await execute(`npx lerna create ${projectName}  --yes`)
```
Comment (fakenickels) [inline newLib.js]: I think as you are already pre-creating the package folder with hygen `lerna create` is not even necessary anymore as it's just creating the package.json 🤔 probably you can use just have a package template in hygen and then we don't even need newLib.js anymore

Comment (jrquadros) [reply newLib.js]: :thinking: but some fields can be a bit difficult to add with a hygen templat  (like author)

Comment (jrquadros) [reply newLib.js]: what do you think @fakenickels ?

Comment (fakenickels) [reply newLib.js]: ah but the author can be `Astrocoders <contact@astrocoders.com>` always!

**Pull request merged**